jsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "@/*": ["src/*"]
+    },
+    "sourceMap": true
+  }
+}
```

Comment:
Not sure if we really need this. (Found it in an online example/doc.) Will follow up.

Reply:
Works with and without it. Should I keep it -- anyone know?

Reply:
I was doing some research, I could be wrong here but I don't see any need for this file for the time being? Especially since everything is working correctly. (https://stackoverflow.com/questions/68675994/what-is-jsconfig-json)

Reply:
Yes, everything seems to work correctly without this file. I just deleted it (in next commit).